packages/principled-forms/src/field/index.ts

```diff
@@ -108,21 +105,19 @@ export class OptionalField<T> implements MinimalField<T> {
 
   constructor({
     type = Type.text,
-    validity = Validity.unvalidated(),
     validators = [],
     value = undefined,
-    eager = true,
   }: OptionalFieldConfig<T> = {}) {
     if (isMaybe(value)) {
+      // Can this be `value.unwrapOr(undefined)`?
```

Comment:
Answer to the comment question here: yes, but only with a type coercion: because `unsafelyUnwrap` is strict about the default value supplied being the same type as that wrapped in a `Maybe`. The type is: `Maybe<T>#unsafelyUnwrap(defaultValue: T)`. Both ways work, though: you can do `value.unwrapOr(undefined as any)` I think. I dislike it b/c type shenanigans are 👎